features/events/syslog/src/test/resources/etc/syslogd-configuration.xml

```diff
@@ -4,7 +4,7 @@
             syslog-port="10514"
             listen-address="127.0.0.1"
             new-suspect-on-message="false"
-            forwarding-regexp="^.*\s(19|20)\d\d([-/.])(0[1-9]|1[012])\2(0[1-9]|[12][0-9]|3[01])(\s+)(\S+)(\s)(\S.+)"
+            forwarding-regexp="^.*\\s*(19|20)\d\d([-/.])(0[1-9]|1[012])\2(0[1-9]|[12][0-9]|3[01])(\s+)(\S+)(\s)(\S.+)"
```

Comment:
Is the extra \ here intentional? I think you just meant to add the asterisk to the \s clause.